### PR TITLE
pyramid

### PR DESCRIPTION
**Pull request by clutchski**
adds integration for the pyramid http tracing.

### CODE REVIEW

File: ddtrace/span.py [hunk 1]
```diff
@@ -255,6 +255,9 @@ def pprint(self):
         lines.extend((" ", "%s:%s" % kv) for kv in sorted(self.meta.items()))
         return "\n".join("%10s %s" % l for l in lines)
 
+    def tracer(self):
```
Comment (palazzem) [inline ddtrace/span.py]: do we need to make it "available" as a ``Span`` public API? or do we need it only for internal instrumentation? in that case I suggest to keep it "internal".

File: ddtrace/contrib/pyramid/trace.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+# project
+import ddtrace
+from ...ext import http, AppTypes
+
+
+def trace_pyramid(config):
+    config.add_tween('ddtrace.contrib.pyramid:trace_tween_factory')
+    # ensure we only patch the renderer once.
+    import pyramid.renderers
```
Comment (palazzem) [inline ddtrace/contrib/pyramid/trace.py]: the import should be here because of a particular import order?

File: ddtrace/span.py (hunk 1 above)
Comment (clutchski) [inline ddtrace/span.py]: I think it's totally fine. So harm at all.

Comment (palazzem) [inline ddtrace/span.py]: 👍

**Pull request merged**